### DISCUSSION TITLE
Benachrichtigung kommt nicht hoch

### DISCUSSION THREAD

**Discussion by Supermario-21** [Zefau/ioBroker.jarvis#1985] [Q&A]
Hallo! Ich komm mit der Benachrichtiung nicht klar! Weis nicht ob es ein Bug ist oder ich zu blöde! Version v3.0.15

Wie kommen die ID und TS zu Stande? Ich verstehe das nicht! Wie kann ich alle Geräte oder nur einzelne Ansprechen? Welche Adresse muss im Browser aufgerufen werden?

**Comment by mcuiobroker** [comment]
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/notifications
In dem obigen Beispiel fehlt timeout.

**Comment by Supermario-21** [comment]
HI! Nach der Anleitung bin ich auch schon vorgegangen! Auch ohne Erfolg! Sorry! Kannst du es mir so erklären, dass ich es auch verstehen kann?

**Comment by mcuiobroker** [comment]
Blockly oder javascript?
Welche Anleitung genau? Da sind mehrere drin.
Es fehlt das "display":"popup" -Objekt und "timeout":5000
Hab noch mal ein Beispiel hinzugefügt:
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/notifications#notwendige-parameter-beispiel
`{
    "title":"TEST", 
    "message":"Test-Nachrichtinhalt",
    "display":"popup",
    "timeout":1500
}`

**Comment by mcuiobroker** [comment]
Man sieht keine Tabzeile? Wo ist die?
Nutze mal screenToGif um es aufzuzeichnen. ->www.screentogif.com
Und die zeit von timeout mal erhöhen auf 20000.

**Comment by mcuiobroker** [comment]
Geh mal in die Browserconsole. (Cmd + Option + J) Stehen da Fehler drin, wenn du die Nachricht schickst?
Für MAC soll es Giphy Capture geben. Probieren?

**Comment by mcuiobroker** [comment]
Die Fehler sind unwichtig. Sonst keine Meldung wenn direkt eine Nachricht versendet wird?

**Comment by mcuiobroker** [comment]
Hast du die PRO Version?

**Comment by Supermario-21** [comment]
Nein!

**Comment by mcuiobroker** [comment]
Die ist dafür notwendig:
https://github.com/Zefau/ioBroker.jarvis/discussions/891

**Comment by Supermario-21** [comment]
Schade das das nicht auf einer Seite bei in der Hilfe steht!

**Comment by mcuiobroker** [comment]
Ich hole es gerade nach.

**Comment by Supermario-21** [comment]
Für was ist dann dieser Schalter? 
![image](https://user-images.githubusercontent.com/104907692/197023052-311cd30e-d07b-4229-ab57-0e177df15780.png)

**Comment by mcuiobroker** [comment]
Der schaltet die Anzeige für den Zustand PRO aus. Ist hier erklärt:
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/allgemein/pro-account#staendige-meldung-welcome-xxx-you-are-pro
und hier:
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/allgemein/pro-account#frontend-anzeige
